macros/WebDriverSidebar.ejs

```diff
@@ -0,0 +1,150 @@
+<section class=Quick_links id=Quick_Links>
+<ol>
+  <li><a href=/<%=locale%>/docs/Web/WebDriver><strong><em>WebDriver</em></strong></a></li>
```

Comment:
`locale` is `undefined`.

It is `env.locale` that contains what you need.

Also, this sidebar macro isn’t localisable.